textile/objects-features.textile

```diff
@@ -99,6 +109,10 @@ h3(#liveobject). LiveObject
 *** @(RTLO3b1)@ Set to an empty map when the @LiveObject@ is initialized, so that any future operation can be applied to this object
 ** @(RTLO3c)@ protected @createOperationIsMerged@ boolean - a flag indicating whether the corresponding @MAP_CREATE@ or @COUNTER_CREATE@ operation has been applied to this @LiveObject@ instance
 *** @(RTLO3c1)@ Set to @false@ when the @LiveObject@ is initialized
+** @(RTLO3d)@ protected @isTombstone@ boolean - a flag indicating whether this object has been tombstoned, i.e. marked for deletion from the objects pool
+*** @(RTLO3d1)@ Set to @false@ when the @LiveObject@ is initialized
+** @(RTLO3e)@ protected @tombstonedAt@ number - a timestamp indicating when this object was tombstoned
```

Comment:
Would `Time` be a more appropriate value for the `tombstonedAt` types?

Reply:
Could we also state that the SDK maintains the invariant that this is non-nil if and only if `isTombstone` is true?

Reply:
Same request as https://github.com/ably/specification/pull/350#discussion_r2322347934 to tighten up the wording a bit